teamfu/.gitignore

```diff
@@ -0,0 +1,62 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
+
+# User-specific files
+*.suo
+*.user
+*.userosscache
+*.sln.docstates
```

Comment:
Can we keep this file for visual studio?**please**

Reply:
@TillCM addressed in https://github.com/TillCM/teamfu/pull/3/commits/3fdefcc6f2ed85d01ee2efd5b2a03f765aa8c35a